index.html

```diff
@@ -22,8 +22,25 @@
   gtag('config', 'G-7960NHDJR9');
 </script>
 </head>
-
 <body>
+    <div class="blur">
+        <div class="mobile">
+            <ul class="menuMenu">
+                    <li class="nav-tag">
+                    <a class="portfolio"  href="#portfolio-sec">Portfolio</a>
+                    </li>
+                    <li class="nav-tag">
+                    <a  href="#about-sec">About</a>
+                    </li>
+                    <li class="nav-tag">
+                    <a  href="#contact-sec">Contact</a>
+                    </li>
+            </ul>
+            <button type="button" class ="hamburger">
```

Comment:
- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that the _hamburger_ and the _close button_ are behaving as buttons when the mouse hovers on them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _hamburger_ and the _close buttons on _mouse hover_ events. We can additionally change the color or opacity of these objects during the hover event.